integration/fuse/src/main/java/alluxio/fuse/OpenFileEntry.java

```diff
@@ -60,6 +63,20 @@ public FileOutStream getOut() {
   }
 
   /**
+   * @return the offset of the most recent write
+   */
+  public long getWriteOffset() {
```

Comment:
What do you think of having `mWriteOffset` track the *next* offset to write to, instead of the latest offset written to? I think this is a more common meaning for "write offset". It also allows us to catch cases where FUSE requests us to write 128KB at offset 0, then 64KB at offset 64KB.

Reply:
that sounds good.